tutorials/tutorial04_meshing/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/presentation/tutorial04_step07_extra_element_ids.md

```diff
@@ -63,6 +64,8 @@
 
 - Pin and Assembly IDs are applied during creations of assemblies and core, respectively.
 
+- [FlexiblePatternGenerator.md] also supports creation of reporting IDs for each unique pin structure (See "Advanced Meshing Tools" section)
```

Comment:
@GiudGiud do you know of any way to reference this section in the slides? I tried
```
See ["Advanced Meshing Tools"](tutorial04_meshing/presentation/index.md#/15) section
```
and that works, where 15 is the slide number of the section, but this will point to the wrong section if the tutorial content is reordered

Reply:
see shortcut links here
https://mooseframework.inl.gov/python/MooseDocs/extensions/core.html

that should work by adding an id to a section

Reply:
Not having luck with this when it comes to the presentation. I added id=sec_name in the header I want to link to and then [#sec_name] in this section, but the link does not work

Reply:
Keeping as `See ["Advanced Meshing Tools"](tutorial04_meshing/presentation/index.md#/15) section` for now. Do you want me to add an issue for this? Referencing other sections with `id= ` doesn't seem to work for presentations

Reply:
ah ok. Yeah let's have an issue

Reply:
https://github.com/idaholab/moose/issues/30297